README.md

```diff
@@ -377,6 +377,15 @@ Example:
 }
 ```
 
+## FQL reserved field names
+Field names may be prepended with special characters to indicate that the field is reserved for a specific purpose.
+
+### Underscore (_)
+Fields prepended with an underscore are reserved for use by FQM consumers.
```

Comment:
I misunderstood the implementation in chat (I thought mod-lists would look for missing records in its results from FQM and inject the missing ones; the approach here works, too, though). Since this actually _is_ being set by FQM, then we should reserve the `_` prefix for use by FQM instead of FQM consumers